README.md

```diff
@@ -69,6 +69,10 @@ reporting the percentage of code covered. Start it with:
 npm run-script coverage
 ```
 
+### Docker
+
+The `docker-start` and `docker-test` scripts are deprecated, and only remain for backwards compatibility. Instead, developers should configure `blubber.yaml` and use [Blubber](https://github.com/wikimedia/blubber) to generate the desired Dockerfile.
+
```

Comment:
Perhaps it would worth adding a working example?

Reply:
in. .pipeline/blubber.yaml

Reply:
thats a good idea, added a working example. let me know if thats what you had in mind?